Doc/library/errno.rst

```diff
@@ -659,12 +659,139 @@ defined by the module.  The specific list of defined symbols is available as
 
    Quota exceeded
 
+
 .. data:: EQFULL
 
    Interface output queue is full
 
    .. versionadded:: 3.11
 
+
+.. data:: ENOMEDIUM
+
+   No medium found
+
```

Comment:
For each constant, please try to check which version added them and add `versionadded`.

Reply:
Emmm. It's difficult, but I'll try to find it.

Reply:
Quick tip: a good way to determine which version added a constant is by using the [git blame of `errnomodule.c`](https://github.com/python/cpython/blame/main/Modules/errnomodule.c). Find the line that defines the constant, click the "blame prior to" button until you find the commit that first added it, then open the commit page. Near the top, just below the commit message, there will be a list of tags that include the commit in their history. The oldest tag listed should be the version that added the constant (barring anything weird happening in the history)